ui/admin/src/app/records/search/brief-view/bio-informations.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'bioInformations'
+})
+export class BioInformationsPipe implements PipeTransform {
+
+  transform(value: any, args?: any): any {
```

Comment:
args is not used in function ?

Reply:
no this has been generated by angular tools, I will remove it.

Reply:
not deleted ?